src/NovaDependencyContainer.php

```diff
@@ -153,6 +153,23 @@ public function resolveForDisplay($resource, $attribute = null)
                 continue;
             }
 
+            if (is_object($resource->{$dependency['property']}) && is_array($dependency['value'])) {
```

Comment:
Could you give an example in which scenario `is_object($resource->{$dependency['property']})` is used?

Reply:
This is used for when a dependency is on a BelongsTo.  I have dependency container that is activated by the selection of a BelongsTo field.  In that scenario, an object is passed here.  You need to compare the id of that object to the passed in array of ids to match.